android-auto-app/build.gradle

```diff
@@ -64,6 +64,6 @@ dependencies {
     implementation "androidx.fragment:fragment-ktx:1.4.1"
 
     // Mapbox Navigation Android Auto SDK Developer Preview
-    implementation("com.mapbox.navigation:ui-androidauto:0.11.0")
+    implementation("com.mapbox.navigation:ui-androidauto:0.12.0")
     implementation("com.mapbox.navigation:ui-dropin:2.8.0-rc.1")
```

Comment:
We're going to need to wait until `2.9.0-alpha.4` is released next week

Reply:
Actually I'm not running into issues here 🤔

Reply:
BTW `2.9.0-alpha.4` is going to be released this week 🚀

Reply:
```
implementation("com.mapbox.navigation:ui-dropin:2.9.0-alpha.3")
```

Ok this is the incompatibility. We cannot upgrade to 2.9.0 yet. But cool! I didn't stop to think if `0.12.0` was compatible with `2.8`, and good to see that it is!

```
2022-09-27 12:48:24.522 4975-4975/com.mapbox.navigation.examples.androidauto E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.mapbox.navigation.examples.androidauto, PID: 4975
    java.lang.NoSuchMethodError: No static method setup$default(Lcom/mapbox/navigation/ui/app/internal/SharedApp;Landroid/content/Context;Lcom/mapbox/navigation/ui/voice/internal/MapboxAudioGuidance;Lcom/mapbox/navigation/ui/maps/internal/ui/RouteAlternativeContract;ILjava/lang/Object;)V in class Lcom/mapbox/navigation/ui/app/internal/SharedApp; or its super classes (declaration of 'com.mapbox.navigation.ui.app.internal.SharedApp' appears in /data/app/~~SsiCGr7_EMJDaJs6wZXXkA==/com.mapbox.navigation.examples.androidauto-QBUyKICRFfn0eB-9Pfkq9A==/base.apk!classes7.dex)
        at com.mapbox.androidauto.MapboxCarApp.setup(MapboxCarApp.kt:51)
        at com.mapbox.navigation.examples.androidauto.ExampleApplication.onCreate(ExampleApplication.kt:23)
        at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1266)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6785)
        at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2134)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.app.ActivityThread.main(ActivityThread.java:7898)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
```

Reply:
Yeah my comments here do not make sense because I thought I pulled the `narko:arv-ui-androidauto-012-changes` branch but didn't.

Reply:
Ok I am replicating your change in a different branch and ran into the error that I was expecting.

But this is specific to the `android-auto-app` because it is an incompatibility with `dropin-ui`

```
Caused by: java.lang.ClassNotFoundException: Didn't find class "com.mapbox.navigation.ui.voice.internal.MapboxAudioGuidance" on path: DexPathList[[dex file "/data/data/com.mapbox.navigation.examples.androidauto/code_cache/.overlay/base.apk/classes4.dex", zip file "/data/app/~~eoFW069spSbSmskx8E4JOg==/com.mapbox.navigation.examples.androidauto-Fz6s_rCRvKpOt59uU4ZL1Q==/base.apk"],nativeLibraryDirectories=[/data/app/~~eoFW069spSbSmskx8E4JOg==/com.mapbox.navigation.examples.androidauto-Fz6s_rCRvKpOt59uU4ZL1Q==/lib/arm64, /data/app/~~eoFW069spSbSmskx8E4JOg==/com.mapbox.navigation.examples.androidauto-Fz6s_rCRvKpOt59uU4ZL1Q==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        	... 40 more
```